.github/workflows/deploy.yml

```diff
@@ -0,0 +1,44 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - jay/website
```

Comment:
I assume this should just reference the main branches "website" directory and not yours?

Reply:
These two changes you commented on shouldn't be committed, they're in the top "revert me" commit. They are changes required to test the change on my fork but would break things on the actual build.